### DISCUSSION TITLE
ise, veya ... ise, veya

### DISCUSSION THREAD

**Discussion by volkantash** [turkce-yazilim-konati/yazilimca#38] [Ölçünler]
...Oradaki karaltıya git bak bakalım:
......araba ise:
..........3 kez ışıkları yak-söndür
......veya bisiklet ise:
..........2 kez ışıkları yak-söndür
......veya motor ise:
..........4 kez ışıkları yak-söndür
......veya:
..........1 kez ışıkları yak-söndür

Gidip baktığınız yerde hem araba hem bisiklet var. Ne yapacaksınız?

**Comment by turkmenistanli** [comment]
"eğer"i mi kaldıracağız?

**Reply by volkantash** [reply]
Eğer yazmak şart mı?

**Comment by turkmenistanli** [comment]
Hayır
